BREAKING_CHANGE_AND_DEPRECATION.md

```diff
@@ -0,0 +1,46 @@
+# Breaking changes and deprecations
+
+Breaking changes are defined as a change to any of the following that causes installation errors or 
+unexpected runtime behavior after upgrading to the next stable minor version of Ratify(cli, plugin,):
+- Verification result and schema
+- Default configuration value
+- User facing plugin interfaces
+
+We do our best to maintain backwards compatibility, however breaking changes may be introduced between version for features that have not yet reached v1.0.0 or only available in RATIFY_EXPERIMENTAL flag.
+
+### Features in Preview
+|     Feature    |  GA Plan |
+|:--------------:|-----:|
+| CRD  |  |  
+| Dynamic plugin |                          |         
+|    HA            |                          |
+ 
+
+## Process for applying breaking changes
+
+- A deprecation notice must be posted as part of a release.
+- The breaking changes are applied two (2) releases after the release in which the deprecation was announced.
```

Comment:
1 question, if feature X is announced to be deprecated in v1.4, and the next release is v2.0, should we remove it in v2.0 or v2.1?

Reply:
since a major version bump indicates breaking change, i think we can do that in v2.0